pywinauto/base_wrapper.py

```diff
@@ -895,7 +906,13 @@ def _type_keys(
             aligned_keys = six.text_type(keys)
 
         # Play the keys to the active window
-        func_to_call(aligned_keys, **kwargs)
+        keyboard.SendKeys(
```

Comment:
Need to call `send_keys` here to avoid DeprecationWarning's for correct `.type_keys` calls.